x/emissions/keeper/msgserver/msg_server_stake.go

```diff
@@ -80,6 +80,15 @@ func (ms msgServer) StartRemoveStake(ctx context.Context, msg *types.MsgStartRem
 		return nil, types.ErrInsufficientStakeToRemove
```

Comment:
You should remove this conditional as it will always be false whenever there's any delegation (related to the purpose of this PR). But you should use the `ErrInsufficientStakeToRemove` error in the conditional you otherwise correctly added below.

Reply:
updated